modules/geo-layers/src/tile-layer/utils.js

```diff
@@ -64,6 +88,16 @@ function getBoundingBox(viewport, zRange, extent) {
   } else {
     bounds = viewport.getBounds();
   }
+  if (!viewport.isGeospatial) {
+    return [
```

Comment:
Should this apply to all viewports?

Reply:
I didn't know enough about the geospatial coordinates so I sequestered it off in an attempt not to change anything from the previous way things worked.  My instinct to check `!viewport.isGeospatial` was that geospatial coordinates could "wrap around" where you have a "left" latitude greater than the "right" latitude of the unprojected viewport.  But I guess if you're using the `extent` prop this is not necessarily the case since `extent` defines a box to begin with.  I can play around a bit to get a feel.

Reply:
Sorry if I have that wrong too about how geospatial coordinates can work.

Reply:
Let's keep it as is for now. I can dig into it.

Reply:
Thanks!